### ISSUE TITLE
Developer portal overview accessibility issues

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#10215]
### Description:

1. Endpoint text is not associated with the label
![image](https://user-images.githubusercontent.com/20179540/110576551-fb7ff580-8186-11eb-917f-6aa8a5aa139b.png)

![image](https://user-images.githubusercontent.com/20179540/110578079-01c3a100-818a-11eb-8d2e-a988d42ad09d.png)

2. IDs used in ARIA and labels must be unique
![image](https://user-images.githubusercontent.com/20179540/110578208-4b13f080-818a-11eb-8139-6d810deffc61.png)

4. need to have HTML5 landmark elements - main
`<main>main content</main>`
5. title should have h1 component
![image](https://user-images.githubusercontent.com/20179540/110581072-dd6ac300-818f-11eb-81ff-2ea4b4c60ab8.png)

6. Heading levels should only increase by one
h1 -Overview 
h2 - PizzaShackAPI 
h3 - card headings
![image](https://user-images.githubusercontent.com/20179540/110581202-12771580-8190-11eb-8724-e52115afaf18.png)

7. API summery should only come in overview. We can remove links from API name and image
![image](https://user-images.githubusercontent.com/20179540/110581474-a2b55a80-8190-11eb-903c-8fe62bfd046f.png)

### Environment details (with versions):
- OS:
- Client:
- Env (Docker/K8s):

---
